package.json

```diff
@@ -55,5 +55,9 @@
         "extends": [
             "@commitlint/config-conventional"
         ]
+    },
```

Comment:
Im not sure why we need that 🤔

Reply:
Normally, I don't we would. However, if I don't include this extension, I cannot commit (which I'm not sure why).

Reply:
which node version are you running?

Reply:
v12.17.0